mint.json

```diff
@@ -84,12 +96,12 @@
       ]
     },
     {
-      "group": "API Playground",
+      "group": "API Reference",
       "pages": [
-        "api-playground/overview",
-        "api-playground/configuration",
-        "api-playground/authentication",
-        "api-playground/openapi"
+        "api-reference/overview",
+        "api-reference/configuration",
+        "api-reference/authentication",
+        "api-reference/openapi"
```

Comment:
I would recommend API Playground over API Reference as `API Reference` is widely used to refer to the API endpoints. Something like this for instance https://www.flipt.io/docs/reference/overview